include/simulation_lifecycle/error.h

```diff
@@ -1,9 +1,16 @@
 #ifndef SIMULATION_LIFECYCLE_ERROR_H
```

Comment:
In the coding conventions provided for the project, Christina adds two underscores "__" before and after header guards. We should probably amend our header guards to match convention?

Reply:
Not sure about that... I usually use the ones provided automatically by CLion.

I'll ask Cristina about that.

Reply:
I am familiar with header guards being written how you are writing them Roman, but maybe Cristina likes them with the two underscored.

Reply:
Cristina answered me that the style guide for this course establishes that header guards are `__LIKE_THAT__`. So we must change it. I'll work on it after the presentations.